src/kimp/kimp.py

```diff
@@ -175,22 +175,27 @@ def run(
         self,
         pattern: Pattern,
         *,
-        depth: int | None = None,
+        bound: int | None = None,
```

Comment:
I would leave the name `depth` here, it is a known option for `krun` and more intuitive than `bound` IMHO

Reply:
Done.

(I used `bound` to indicate that it is the upper bound of the number of rule applications, but I agree consistency matters here.)